ci-operator/step-registry/openshift-qe/etcd-density-db-quota-pressure/openshift-qe-etcd-density-db-quota-pressure-commands.sh

```
@@ -0,0 +1,59 @@
+ES_HOST="search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
+ES_PASSWORD=$(cat "${ES_SECRETS_PATH}/password")
+ES_USERNAME=$(cat "${ES_SECRETS_PATH}/username")
+if [ -e "${ES_SECRETS_PATH}/host" ]; then
+    ES_HOST=$(cat "${ES_SECRETS_PATH}/host")
+fi
+UUID=$(uuidgen)
+
+REPO_URL="https://github.com/cloud-bulldozer/e2e-benchmarking"
+LATEST_TAG=$(git ls-remote --tags "${REPO_URL}.git" | awk -F'refs/tags/' '{print $2}' | grep -v '\^{}' | sort -V | tail -n1)
+TAG_OPTION="--branch $(if [ "$E2E_VERSION" == "default" ]; then echo "$LATEST_TAG"; else echo "$E2E_VERSION"; fi)"
+git clone $REPO_URL $TAG_OPTION --depth 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)